packages/spark-core/base/_links.scss

```diff
@@ -18,8 +18,8 @@
   &:hover,
   &:active,
   &:focus {
-    color: $link-underline-color;
-    border-bottom: $link-underline-width solid $link-underline-color;
+    color: $link-hover-color;
+    border-bottom: $link-underline-width solid $link-hover-color;
   }
 }
 
```

Comment:
I can't comment on the line because it wasn't changed in this PR, but the border being set on standalone links needs to be more specific. Having a transparent border around the whole link causes the hover state to be jank
<img width="410" alt="screen shot 2018-12-18 at 4 19 36 pm" src="https://user-images.githubusercontent.com/308696/50183670-c30f4b00-02e0-11e9-8821-1c069fa70ada.png">